src/core/ext/filters/client_channel/client_channel.cc

```diff
@@ -1128,26 +1127,21 @@ static bool pick_callback_start_locked(grpc_call_element* elem) {
       initial_metadata_flags &= ~GRPC_INITIAL_METADATA_WAIT_FOR_READY;
     }
   }
-  const grpc_lb_policy_pick_args inputs = {
+  calld->pick.initial_metadata =
       calld->initial_metadata_batch->payload->send_initial_metadata
           .send_initial_metadata,
```

Comment:
the trailing comma should be a semicolon.

Reply:
Doh!  Thanks for catching this.  Fixed.